src/aeb_fate_generate_ops.erl

```diff
@@ -104,7 +104,7 @@ ops_defs() ->
     , { 'ORIGIN',       16#5b,    1,  false,   3, [a],              origin, "Arg0 := Address of contract called by the call transaction."}
     , { 'CALLER',       16#5c,    1,  false,   3, [a],              caller, "Arg0 := The address that signed the call transaction."}
     , { 'GASPRICE',     16#5d,    1,  false,   3, [a],            gasprice, "Arg0 := The current gas price."}
-    , { 'BLOCKHASH',    16#5e,    1,  false,   3, [a],           blockhash, "Arg0 := The current blockhash."} %% TODO: Do we support has at height?
+    , { 'BLOCKHASH',    16#5e,    2,  false,   3, [a, a],        blockhash, "Arg0 := The blockhash at height."}
```

Comment:
Any? Or one of the most recent 256 ones?

Can height be below genesis i.e. negative? If so, what is behaviour?

Any special behavioury in state channels - on- (force progress) or off-chain?

Ref https://github.com/aeternity/protocol/blame/f1a1ca14a7f0c95aaeba765b8bec76c77f358493/contracts/aevm.md#L81-L87

Reply:
I don't think these strings are supposed to describe the full semantics of the instructions, but if they are, we need a total overhaul at some point.

I'll leave it as it is for now.